### ISSUE TITLE
Fix warning in mclearn.viz.plot_hex_map

### ISSUE THREAD

**Issue by jakubnabaglo** [chengsoonong/mclass-sky#149]
```
cbook.py:136: MatplotlibDeprecationWarning: The axisbg attribute was deprecated in version 2.0. Use facecolor instead.
  warnings.warn(message, mplDeprecation, stacklevel=1)
```

**Comment by jakubnabaglo** [comment]
Closed by #150.

**Comment by chengsoonong** [comment]
Resolved in 598b13a
